tests/testMatchOptimisticB.py

```diff
@@ -132,8 +133,9 @@ def computePosRefCatalog(self, sourceCat):
         refCat = afwTable.SimpleCatalog(minimalPosRefSchema)
         for source in sourceCat:
             refObj = refCat.addNew()
-            refObj.set("coord", source.get("coord"))
-            refObj.set("centroid", source.getCentroid())
+            refObj.setCoord(source.getCoord())
+            refObj.set("centroid_x", source.getX())
+            refObj.set("centroid_y", source.getY())
```

Comment:
I disagree that it's cleaner.  It's a lot of setup for a one-liner, and while it'd be more efficient if it was done repeatedly in production code, I don't think it matters in test code.